service.subtitles.napisy24pl/resources/lib/NapisyUtils.py

```diff
@@ -0,0 +1,345 @@
+# -*- coding: utf-8 -*-
+from http.cookiejar import LWPCookieJar
+from urllib.request import HTTPCookieProcessor, build_opener
+from urllib.parse import urlencode
+import json
+import os
+import re
+import shutil
+import unicodedata
+import zlib
+import bs4
+
+import xbmc
+import xbmcvfs
+import xbmcaddon
+
+__addon__ = xbmcaddon.Addon()
+__version__ = __addon__.getAddonInfo('version')  # Module version
+__scriptname__ = __addon__.getAddonInfo('name')
+__language__ = __addon__.getLocalizedString
+__profile__ = xbmc.translatePath(__addon__.getAddonInfo('profile'))
+__temp__ = xbmc.translatePath(os.path.join(__profile__, 'temp', ''))
+
+regexHelper = re.compile('\W+', re.UNICODE)
+
+
+def normalizeString(str):
+    return unicodedata.normalize('NFKD', str)
+
+
+def log(msg):
+    xbmc.log((u"### [%s] - %s" % (__scriptname__, msg)), level=xbmc.LOGDEBUG)
+
+
+def notify(msg_id):
+    xbmc.executebuiltin((u'Notification(%s,%s)' % (__scriptname__, __language__(msg_id))))
```

Comment:
xbmcgui.Dialog().notification() please. Try to use the python api as much as possible instead of the builtin

Reply:
What is the alternative api?

Reply:
https://codedocs.xyz/xbmc/xbmc/group__python___dialog.html#ga27157f98dddb21b44cc6456137977aa2